README.md

```diff
@@ -14,20 +15,174 @@ _class:
 
 # Jak psát moderní učebnice
 
-Miroslav Suchý, Sergei Petrosian, Lukáš Růžička:  
+[Miroslav Suchý](mailto:msuchy@redhat.com), [Sergei Petrosian](mailto:spetrosi@redhat.com), [Lukáš Růžička](mailto:lruzicka@redhat.com)
+
+## Učebnice je klasika
+
+![bg right](img/slabikar.jpg)
+
+## Učebnice je klasika?
+
+![bg right fit](img/baltik.jpg)
+
+## Učebnice je klasika?
+
+Trvanlivost a odolnost materiálu, mnohdy desítky let.
+
+![bg right fit](img/not-found.png)
+
+## Jenže...
+
+Napsat učebnici trvá rok.
+
+Vydat učebnici trvá taky rok.
+
+V informatice jsou dva roky věčnost. Ale platí to i pro jiné obory.
+
+
+## Takže...
+
+V den vydání je učebnice prakticky dva roky stará.
+
+## Errata
+
+*Drobné aktualizace v kapitole 29 – Patologie GIT*
+
+*Str. 509 a dále. Po odevzdání podkladů pro učebnici byla vydána nová WHO klasifikace nádorů trávicího systému, ve které byl „sesilní serrated adenom“ (SSA) přejmenován na „sesilní serrated lézi“ (SSL). Praktický dopad to má ten, že opět (po přibližně desetiletém intermezzu) platí staré pravidlo „každý adenom obsahuje dysplázie“ (protože SSA byl v tomto výjimkou).*
+
+## Chuť k opravám
+
+Různé formáty: Rukopis -> Nakladatelství -> Tiskárna
+
+*HTML*, *PDF* a online publikace zpravidla nejsou původní zdroj.
+
+Nízká bariéra pro opravy a úpravy.
+
+## Požadavky na moderní učebnici
+
+- Tvorba a publikování lze dělat současně.
+
+- Učebnice jsou dostupné všem a zdarma (nebo za minimální poplatek).
+
+- Na vytváření učebnic může spolupracovat množství nezávislých autorů, dokonce i sami studenti.
+
+- Obsah lze měnit, přizpůsobovat a neustále vylepšovat.
+
+- Obsah lze jednoduše převést.
+
+## Komunitní dílo (+)
+
+- Jeden člověk vše neobsáhne.
+
+- Možnost použít jenom část díla.
+
+- Online přítomnost a aktuálnost.
+
+## Komunitní dílo (-)
+
+- Autorské poplatky.
+
+- Neukážete to babičce.
+
+## <!--fit--> Jak na to?
+
+<!--
+Tady si předáme slovo
+-->
+
+Je mnoho možností.
+
+Ukážeme si jednu z mnoha.
+
+## <!--fit--> Psaní dokumentace ve formátu "Docs-as-Code"
+
+Principy Docs-as-Code:
+
+- správa verzí (Git)
+- umožňuje, aby na projektu spolupracovalo víc lidi
+- kontrola změn, kterou autor může delegovat na spolupracovníky
+- asynchronní spolupráce (každý, kdy má čas)
+- automatické testování, generování, a publikace materiálů
+- psaní ve značkovacích jazycích (Markdown, AsciiDoc, LaTeX, atd.) namísto WYSIWYG (LibreOffice, MS Word)
+
+## Zápis jazyka Markdown
+
+```md
+# Název
+## Kapitola 1. Tučné písmo, kurzíva, a monospace
+Tento řádek obsahuje **tučné písmo**, *kurzívu*, a `monospace`
+
+### Podkapitola 1.1. Číslovaný seznam pomocí čísel
+1. Řádek 1
+2. Řádek 2
+
+## Kapitola 2. Seznam odrážek pomocí pomlček
+
+* Řádek 1
```

Comment:
Tady jsem si všiml, že to `pandoc` buildil jako jednoduchý a ne vnořený seznam. Tak jsem to přepsal na tu druhou variantu, jestli se to chytne. Prověřit.

Reply:
Taky jsem to vsiml, dekuji za opravu, s hvezdicky to funguje jak ma